### PR TITLE
Feature/#505 add quiz flowdiagram

### PR DESCRIPTION
**Pull request by ritik307**
Added Pop Quiz in the `docs/seq-design`

Fixes #505 

#### Changes done:
- [x] Pop Quiz in `flow-diagram.md`.

#### Screenshots
Flowchart

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by ritik307** [comment]
@Shivansh2407  plz review my PR

File: docs/seq-design/flow-diagrams.md [hunk 1]
```diff
@@ -47,3 +47,19 @@ More information about ASM charts can be found in:
 ## References
 
 {% bibliography --cited --file books %}
+
+{:.quiz}
+1. A processing step is denoted by a `________` ?
+   1. rectangular box
```
Comment (manjotsidhu) [inline docs/seq-design/flow-diagrams.md:53]: First letter must be capital of every option

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by ritik307** [comment]
@Shivansh2407 Done! Plz review

**Comment by manjotsidhu** [comment]
> @Shivansh2407 Done! Plz review

Remove unnecessary changes please.

**Comment by ritik307** [comment]
Done!

**Review by manjotsidhu — CHANGES_REQUESTED** [review]
Remove Unnecessary quotes like ```

**Pull request merged**